app/views/providers/income_summary/_income_type_item.html.erb

```diff
@@ -8,29 +8,24 @@
 <% error_message = error.present? ? error.first : "" %>
 
 <li id="<%= name.to_s %>" class="<%= form_group_error_class %>">
-  <h2 class="app-task-list__section">
+  <h2 class="display-table govuk-heading-m">
     <% if display_number %>
-      <span class="app-task-list__section-number"><%= number %>. </span>
+      <span class="sort-transactions__section-number"><%= number %>. </span>
     <% else %>
-      <span class="app-task-list__section-number"><%= "" %> </span>
+      <span class="sort-transactions__section-number"><%= "" %> </span>
     <% end %>
 
     <%= t("transaction_types.names.providers.#{name}") %>
-    <% if I18n.exists? "transaction_types.hints.#{name}" %>
-      <div class="govuk-hint" style="margin-bottom: 0px">
-        <%= t("transaction_types.hints.#{name}") %>
-      </div>
-    <% end %>
     <% if error_message %>
       <p id='<%= "error-#{name}" %>' class="govuk-error-message govuk-!-padding-top-2"><%= error_message %></p>
     <% end %>
   </h2>
-  <%= content_tag(:div, class: "app-task-list__items", id: "list-item-#{name}") do %>
+  <%= content_tag(:div, class: "govuk-!-padding-bottom-9 govuk-!-padding-left-0", id: "list-item-#{name}") do %>
```

Comment:
nit: can we remove `content_tag` and have just a `<div>` instead?
It seems redundant here to ruse the rails tag helper when we can write the html tag.